package.json

```diff
@@ -62,11 +62,12 @@
   },
   "peerDependencies": {
     "react": "^0.14.9 || ^15.3.0",
-    "prop-types": "^15.5.0",
+    "prop-types": "^15.5.7",
     "react-redux": "^4.0.0 || ^5.0.0",
     "redux": "^3.5.2"
   },
   "dependencies": {
+    "prop-types": "^15.5.7",
```

Comment:
If it's already in `dependencies` it will be installed anyway, so no need for having it also in `peerDependencies` and `devDependencies`. I guess the 3rd point was causing this confusion :)